From: Xuewen Wang <wangxuewen@kylinos.cn>
Subject: mm/damon: remove trailing semicolons after function definitions
Date: Tue, 21 Jul 2026 06:53:31 -0700

Three function definitions terminate with '};' instead of '}', which is
unnecessary and inconsistent with kernel coding style:

- damon_pa_initcall() in paddr.c
- damon_va_initcall() in vaddr.c
- damos_get_some_mem_psi_total() in core.c

No functional change intended.

Link: https://lore.kernel.org/20260721135333.241106-1-sj@kernel.org
Signed-off-by: Xuewen Wang <wangxuewen@kylinos.cn>
Reviewed-by: SJ Park <sj@kernel.org>
Signed-off-by: SJ Park <sj@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/damon/core.c  |    2 +-
 mm/damon/paddr.c |    2 +-
 mm/damon/vaddr.c |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

--- a/mm/damon/core.c~mm-damon-remove-trailing-semicolons-after-function-definitions
+++ a/mm/damon/core.c
@@ -2791,7 +2791,7 @@ static u64 damos_get_some_mem_psi_total(
 static inline u64 damos_get_some_mem_psi_total(void)
 {
 	return 0;
-};
+}
 
 #endif	/* CONFIG_PSI */
 
--- a/mm/damon/paddr.c~mm-damon-remove-trailing-semicolons-after-function-definitions
+++ a/mm/damon/paddr.c
@@ -447,6 +447,6 @@ static int __init damon_pa_initcall(void
 	};
 
 	return damon_register_ops(&ops);
-};
+}
 
 subsys_initcall(damon_pa_initcall);
--- a/mm/damon/vaddr.c~mm-damon-remove-trailing-semicolons-after-function-definitions
+++ a/mm/damon/vaddr.c
@@ -959,7 +959,7 @@ static int __init damon_va_initcall(void
 	if (err)
 		return err;
 	return damon_register_ops(&ops_fvaddr);
-};
+}
 
 subsys_initcall(damon_va_initcall);
 
_
